convert-to-pdf.sh

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 
 npx mr-pdf --initialDocURLs="http://localhost:3000" --contentSelector="article" --paginationSelector=".pagination-nav__item--next > a" --excludeSelectors=".margin-vert--xl a,.theme-edit-this-page,.tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh,.tocCollapsibleButton_2O1e" --coverImage="https://download.logo.wine/logo/Seneca_College/Seneca_College-Logo.wine.png" --coverTitle="Introduction to C Programming" --outputPDFFilename "Introduction-to-C.pdf"
+mv "./Introduction-to-C.pdf" "./static"
```

Comment:
Instead of `mv`'ing this, should we alter line 3 to use `--outputPDFFilename "./static/Introduction-to-C.pdf"`?  If it lets us give a path vs. just the filename, that would do it in one step.

Reply:
I made the script one line doing the same job and my test result is good. Please review the updated. Thanks